decoders/connector/netvox/ra02g/v1.0.0/payload.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-restricted-syntax */
+
+/*
+** RA02G Custom Payload Decoder for TagoIO 
+** Parses sensor data such as alarms, battery, and environmental readings.
+**
+*/
+
+const ignore_vars = [];
+
```

Comment:
Please add unit tests in order to make sure the connector is working. Please use the Netvox R312a as an example.